XPlat/AzureMessaging/iOS/source/ApiDefinition.cs

```diff
@@ -158,3 +158,78 @@ public partial interface SBTokenProvider
 	}
 }
 
+namespace WindowsAzure.Messaging.NotificationHubs
+{
+	[BaseType (typeof (NSObject))]
+ 	public partial interface MSNotificationHub 
+ 	{
+ 		[Static, Export ("initWithConnectionString:hubName:")]
+ 		void Init (string connectionString, string hubName);
+
+        [Static, Export ("didRegisterForRemoteNotificationsWithDeviceToken:")]
+ 		void DidRegisterForRemoteNotifications (NSData deviceToken);
+
+ 		[Static, Export ("didFailToRegisterForRemoteNotificationsWithError:")]
+ 		void DidFailToRegisterForRemoteNotifications (NSError error);
+
+ 		[Static, Export ("didReceiveRemoteNotification:")]
+ 		bool DidReceiveRemoteNotification (NSDictionary userInfo);
+
+ 		[Static, Export ("setDelegate:")]
+ 		void SetDelegate ([NullAllowed] IMSNotificationHubDelegate nubDelegate);
+
+        [Static, Export("isEnabled")]
+		bool IsEnabled();
+
+		[Static, Export("setEnabled:")]
+		bool SetEnabled(bool isEnabled);
+
+		[Static, Export("getPushChannel")]
+		string GetPushChannel();
+
+		[Static, Export("getInstallationId")]
+		string GetInstallationId();
+
+		[Static, Export ("addTag:")]
+ 		bool AddTag (string tag);
+
+ 		[Static, Export ("addTags:")]
+ 		bool AddTags (NSArray<NSString> tags);
+
+ 		[Static, Export ("removeTag:")]
+ 		bool RemoveTag (string tag);
+
+ 		[Static, Export ("removeTags:")]
+ 		bool RemoveTags (NSArray<NSString> tags);
+
+ 		[Static, Export ("getTags")]
+ 		NSArray<NSString> GetTags ();
+
+ 		[Static, Export ("clearTags")]
+ 		void ClearTags ();
+ 	}
+
+ 	[BaseType(typeof(NSObject))]
+    [Model, Protocol]
+    public interface MSNotificationHubDelegate 
+ 	{
+        [Abstract, Export ("notificationHub:didReceivePushNotification:")]
+     	void DidReceivePushNotification(MSNotificationHub notificationHub, MSNotificationHubMessage message);
```

Comment:
This has a UICompletionHandler in the preview version but not here, is there a reason for that?

Reply:
Fixed, seems there were changes in api and bindings missed some updates.